### ISSUE TITLE
SMS Reminders in DUP

### ISSUE THREAD

**Issue by kgorana** [bcgov/parks-reso-public#278]
**Description**
As a ….

**Acceptance Criteria:**

**Notes**

**Comment by MarshallHenson** [comment]
@JLWade I spoke with Nicole and we finalized some copy for both the reservation page and the text message. She is going to pass them along to Chris

**Comment by manuji** [comment]
Merging this PR to the Main branch is blocked by the pending PROD deployment. I will let @Christopher-walsh22 know when things are all sorted and ready for new code merging. 

FYI @ashley-dhillon

**Comment by Christopher-walsh22** [comment]
@manuji  has confirmed the environment is clear and this is ready to be merged. I have also updated the environments (Dev + Test+ Prod) with the needed parameters for using gc notify sms features.

**Comment by manuji** [comment]
Tested on TEST: Failed
* As a user who had a pass for the next day did not get the SMS at 4 pm.

**Comment by manuji** [comment]
Tested on TEST: **Pending**

* verified that the SMS is received at 4 pm the day before the visit if the phone number is provided and if the pass has not been canceled. 
* There is a small issue with the SMS format below; the pass type (am, pm, all-day) should be displayed in lower case and "DAY" should be displayed as "all-day". 
* Also, the sentence starts with "if..." should be started in a new line. 



![Image](https://github.com/bcgov/parks-reso-public/assets/65190263/87e8c2f7-ae80-41e6-9e51-340ce153b71f)

**Comment by Christopher-walsh22** [comment]
@manuji  I have updated the template in GCNotify and have a PR [code-review](https://github.com/bcgov/parks-reso-api/pull/335) addressing these issues :)

**Comment by JLWade** [comment]
@Christopher-walsh22 can you add estimated story points to this one

**Comment by JLWade** [comment]
Good catch @manuji I think we should resolve but not urgent - can you make a new ticket for that an am (to replace a am) and also to capitalize AM and PM? add it to the backlog assigned to me and I can move it into a future sprint.

**Comment by JLWade** [comment]
Moving this one to done
